### PR TITLE
OSDOCS-12303-phase-4: Reorg of the networking operator docs

### PR DESCRIPTION
**Pull request by dfitzmau**
Purpose:

This PR sets up the final addition to a consolidated networking Operator shared directory. I have created xrefs to help with the reorg and to orientate our customers that bit better to the new doc structure. Redirects are also been created.

Version(s):
4.16+

Issue:
[OSDOCS-12303](https://issues.redhat.com/browse/OSDOCS-12303)

Link to docs preview:
* [Networking Operators - Core](https://85441--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/networking_operators/sr-iov-operator/installing-sriov-operator)
* [Hardware networks - Core](https://85441--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/hardware_networks/about-sriov)
* [Networking Operators ROSA](https://85441--ocpdocs-pr.netlify.app/openshift-rosa/latest/networking/networking_operators/aws-load-balancer-operator)
* [Networking Operators Dedicated](https://85441--ocpdocs-pr.netlify.app/openshift-dedicated/latest/networking/networking_operators/dns-operator)

### CODE REVIEW

File: networking/hardware_networks/about-sriov.adoc [hunk 1]
```diff
@@ -33,11 +35,17 @@ You can enable SR-IOV on a node by using the following command:
 $ oc label node <node_name> feature.node.kubernetes.io/network-sriov.capable="true"
 ----
 
+[discrete]
+[role="_additional-resources"]
+[id="additional-resources-about-sr-iov_{context}"]
+== Additional resources
+
+* xref:../../networking/networking_operators/sr-iov-operator/installing-sriov-operator.adoc#installing-sriov-operator[Installing the SR-IOV Network Operator].
```
Comment (kcarmichael08) [inline networking/hardware_networks/about-sriov.adoc:43]:
```suggestion
* xref:../../networking/networking_operators/sr-iov-operator/installing-sriov-operator.adoc#installing-sriov-operator[Installing the SR-IOV Network Operator]
```

File: networking/metallb/metallb-configure-address-pools.adoc [hunk 2]
```diff
@@ -6,8 +6,9 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-As a cluster administrator, you can add, modify, and delete address pools.
-The MetalLB Operator uses the address pool custom resources to set the IP addresses that MetalLB can assign to services. The namespace used in the examples assume the namespace is `metallb-system`.
+As a cluster administrator, you can add, modify, and delete address pools. The MetalLB Operator uses the address pool custom resources to set the IP addresses that MetalLB can assign to services. The namespace used in the examples assume the namespace is `metallb-system`.
+
+For more information about how to install the MetalLB Operator, see xref:../../networking/networking_operators/metallb-operator/about-metallb.adoc#about-metallb[About MetalLB and the MetalLB Operator]
```
Comment (kcarmichael08) [inline networking/metallb/metallb-configure-address-pools.adoc:11]:
```suggestion
For more information about how to install the MetalLB Operator, see xref:../../networking/networking_operators/metallb-operator/about-metallb.adoc#about-metallb[About MetalLB and the MetalLB Operator].
```

File: networking/networking_operators/external_dns_operator/external-dns-operator-release-notes.adoc [hunk 3]
```diff
@@ -20,7 +20,7 @@ These release notes track the development of the External DNS Operator in {produ
 
 The following advisory is available for the External DNS Operator version 1.3.0:
 
-*link:https://access.redhat.com/errata/RHEA-2024:8550[RHEA-2024:8550 Produce Enhancement Advisory]
+* link:https://access.redhat.com/errata/RHEA-2024:8550[RHEA-2024:8550 Produce Enhancement Advisory]
```
Comment (kcarmichael08) [inline networking/networking_operators/external_dns_operator/external-dns-operator-release-notes.adoc:23]:
```suggestion
* link:https://access.redhat.com/errata/RHEA-2024:8550[RHEA-2024:8550 Product Enhancement Advisory]
```

**Review by kcarmichael08 — COMMENTED** [review]
Checked all the cross-refs and everything looks good! Left a question and a few tiny things I noticed. Great job!

File: _topic_maps/_topic_map.yml [hunk 4]
```diff
@@ -1418,16 +1418,16 @@ Topics:
...
-#     Dir: sr-iov-operator
-#     Distros: openshift-enterprise,openshift-origin
-#     Topics:
-#     - Name: Installing the SR-IOV Operator
-#       File: installing-sriov-operator
-#     - Name: Configuring the SR-IOV Operator
-#       File: configuring-sriov-operator
-#     - Name: Uninstalling the SR-IOV Operator
-#       File: uninstalling-sriov-operator
+  - Name: SR-IOV Operator
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply _topic_maps/_topic_map.yml:1421]: Thanks. Very good point. Updated!

**Pull request merged**